cookbooks/travis_ci_ruby/attributes/default.rb

```diff
@@ -1,11 +1,11 @@
-override['rvm']['default'] = '2.2.5'
+override['rvm']['default'] = '2.2.6'
 override['rvm']['rubies'] = %w(
   1.9.2
   1.9.3
   2.0.0
   2.1.10
-  2.2.5
-  2.3.1
```

Comment:
@adelcambre I love the idea of smooth transitions.  The maintenance overhead hurts, in my experience, and we have a history of promoting `group: stable` with gaps of 4-9 months.  Tightening up the frequency of `group: stable` promotions also sounds good to me, but we have pressure in both directions from various customers.  I guess what I'm really trying to say here is that we have many variables to consider, and that I prefer to keep the preinstalled versions to a minimum, and fairly close to latest available, with caveats for how different languages do releases and such.  I realize this probably sounds insensitive 😐 ...